Stripe/STPAnalyticsClient.m

```diff
@@ -140,16 +149,38 @@ - (NSArray *)productUsage {
     return productUsage ?: @[];
 }
 
+- (NSDictionary *)productUsageDictionary {
+    NSMutableDictionary *productUsage = [NSMutableDictionary new];
+
+    NSString *uiUsageLevel = nil;
+    if ([self.apiUsage containsObject:NSStringFromClass([STPPaymentContext class])]) {
+        uiUsageLevel = @"full";
+    }
+    else if (self.apiUsage.count == 1
+             && [self.apiUsage containsObject:NSStringFromClass([STPPaymentCardTextField class])]) {
+        uiUsageLevel = @"card_text_field";
+    }
+    else if (self.apiUsage.count > 0) {
+        uiUsageLevel = @"partial";
+    }
+    else {
+        uiUsageLevel = @"none";
+    }
+    productUsage[@"ui_usage_level"] = uiUsageLevel;
+    productUsage[@"product_usage"] = [self productUsage];
+
+    return productUsage.copy;
+}
+
 - (void)logTokenCreationAttemptWithConfiguration:(STPPaymentConfiguration *)configuration
                                        tokenType:(NSString *)tokenType {
     NSDictionary *configurationDictionary = [self.class serializeConfiguration:configuration];
     NSMutableDictionary *payload = [self.class commonPayload];
     [payload addEntriesFromDictionary:@{
                                         @"event": @"stripeios.token_creation",
                                         @"token_type": tokenType ?: @"unknown",
-                                        @"apple_pay_enabled": @([Stripe deviceSupportsApplePay]),
```

Comment:
I moved it to the common payload below, so it will be attached on all analytics, not just token creations. Seems to make more sense since that has all the device/app specific stuff.